src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java

```diff
@@ -0,0 +1,43 @@
+package uk.ac.ic.wlgitbridge.snapshot.base;
+
+import com.google.gson.JsonElement;
+import uk.ac.ic.wlgitbridge.git.exception.SnapshotAPIException;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MissingRepositoryException extends SnapshotAPIException {
+
+    public static final String GENERIC_REASON =
+        "This Overleaf project currently has no git access.\n" +
+        "\n" +
+        "If you think this is an error, contact support at support@overleaf.com.";
```

Comment:
This is OK, but we usually say "If this problem persists, please contact ....", so probably best to be consistent.